templates/content/blank/NewApp/NewApp.fsproj

```diff
@@ -5,11 +5,6 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
This causes XamlC compiler errors in VS for Mac, so removing it.

Reply:
ok, thanks!